### DISCUSSION TITLE
PCL的正常主题解锁方式是啥

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3142] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/116283787/4ba642e5-25b2-42b6-aeb8-7db0f15f8be7)
如图，混乱黄是由点击n次PCL的Logo图标解锁的，那剩下一个主题的正常解锁方式呢？（已知剩下一个主题名为“死机蓝”且在注册表项中名为“ThemeDeathBlue”）

**Comment by MoYuan-CN** [comment]
尝试让 PCL 死机？

**Reply by allMagicNB** [reply]
> 尝试让 PCL 死机？

怎么死机……我翻遍了整个源代码都没找到主题解锁函数……

**Reply by MoYuan-CN** [reply]
或许尝试下在运行 PCL 的时候电脑死机....
记得用一个无害的方式

**Comment by kong-que-dnf123** [comment]
这个是PCL几版本啊🤔

**Reply by allMagicNB** [reply]
PCL1啊.jpg

**Reply by MoYuan-CN** [reply]
1.0.9（可见于上方蓝色背景的右下角）

**Comment by WForst-Breeze** [comment]
源码：https://github.com/LTCatt/PCL1/blob/master/Plain%20Craft%20Launcher/formBlueScreen.xaml.vb

**Reply by allMagicNB** [reply]
看得懂又看不太懂……
我甚至不知道变量干啥用的

**Comment by ZmdNotFound** [comment, marked as answer]
我触发了。
方法：打开调试模式，让PCL输出大量调试日志。之后会触发一次假蓝屏（错误代码：PCL_HINTS_OVERFLOW），等待进度条跑完就会解锁。
建议：可以通过下载最新版本游戏来输出大量日志，因为版本更新后之前的API无法使用导致PCL大量报错。

**Reply by allMagicNB** [reply]
可见早期PCL的彩蛋真的多（

**Reply by iPodtouch6557** [reply]
一些普通的开发者总是喜欢把周边功能先做了
亲身体验

**Reply by wuliaodexiaoluo** [reply]
> 我触发了。 方法：打开调试模式，让PCL输出大量调试日志。之后会触发一次假蓝屏（错误代码：PCL_HINTS_OVERFLOW），等待进度条跑完就会解锁。 建议：可以通过下载最新版本游戏来输出大量日志，因为版本更新后之前的API无法使用导致PCL大量报错。

反复点才行

**Reply by allMagicNB** [reply]
> 反复点才行

都一个月了就别挖坟了……

**Comment by wuliaodexiaoluo** [comment]
> 可见早期PCL的彩蛋真的多（

话说键盘触发行不行

**Reply by Frepidation** [reply]
> > 可见早期PCL的彩蛋真的多（
> 
> 话说键盘触发行不行

不算pcl触发，算函数触发的
